packages/node-client/src/NodeClient.ts

```diff
@@ -185,6 +189,10 @@ class NodeClient implements INodeClient {
   async estimateHandlePaymentGasOverride(
     estimateHandlePaymentTxGasDto: EstimateHandlePaymentTxGasDto
   ): Promise<EstimateGasResponse> {
+    const byPass = process.env.BY_PASS_ESTIMATION
+    if ( byPass === 'true' ){
```

Comment:
and then this var also changes skipGasEstimation or skipHandlerPaymentGasEstimation